bedrock/products/tests/test_helper_misc.py

```diff
@@ -601,3 +601,27 @@ def test_vpn_12_month_saving(self):
             u'<span class="js-vpn-saving-display" data-price-usd="Save 50%" '
             u'data-price-euro="Save 50%" data-price-chf="Save 45%">Save 50%</span>')
         self.assertEqual(markup, expected)
+
+
+@override_settings(DEV=True)
+class TestVPNProductReferralLink(TestCase):
+    rf = RequestFactory()
+
+    def _render(self, referral_id, page_anchor, link_text, class_name, optional_attributes):
+        req = self.rf.get('/')
+        req.locale = 'en-US'
+        return render("{{{{ vpn_product_referral_link('{0}', '{1}', '{2}', '{3}', {4}) }}}}".format(
+                      referral_id, page_anchor, link_text, class_name, optional_attributes),
+                      {'request': req})
+
+    def test_vpn_product_referral_link(self):
+        """Should return expected markup"""
+        markup = self._render(referral_id='navigation', page_anchor='#pricing',
+                              link_text='Get Mozilla VPN', class_name='mzp-t-secondary mzp-t-md',
+                              optional_attributes={'data-cta-text': 'Get Mozilla VPN',
+                                                   'data-cta-type': 'button'})
+        expected = (
+            u'<a href="/products/vpn/#pricing" class="mzp-c-button mzp-t-product '
```

Comment:
@pmac not urgent - any idea why this test passes locally, but not in CI? 

The problem seems to be because in the helper I'm using `reverse('products.vpn.landing')` to generate the link URL, and whilst this returns  `/products/vpn/` for me locally, when CircleCI runs it returns with the locale `/en-US/products/vpn/`. How can I make this consistent?

Reply:
It's probably because the test environment has an active locale at that point. You should be able to ensure this is the case using the following for the _render method:

```python
    def _render(self, referral_id, page_anchor, link_text, class_name, optional_attributes):
        with self.activate('en-US'):
            req = self.rf.get('/')
            req.locale = 'en-US'
            return render("{{{{ vpn_product_referral_link('{0}', '{1}', '{2}', '{3}', {4}) }}}}".format(
                      referral_id, page_anchor, link_text, class_name, optional_attributes),
                      {'request': req})
```

Reply:
if I'm right then it should always have the `/en-US` prefix.

Reply:
This worked, thank you!